extensions/policy-referring-connector/src/main/java/de/sovity/edc/extension/policy/functions/AbstractReferringConnectorValidation.java

```diff
@@ -30,38 +30,46 @@
 
 import java.util.Objects;
 
+import static de.sovity.edc.extension.policy.ReferringConnectorValidationExtension.BUSINESS_PARTNER_NUMBER_CONSTRAINT_KEY;
+
 /**
  * Abstract class for referringConnector validation. This class may be inherited from the EDC
  * policy enforcing functions for duties, permissions and prohibitions.
  */
 public abstract class AbstractReferringConnectorValidation {
```

Comment:
We don't need an abstract class, we only need the permission for now 


Also see AlwaysTruePolicyService#registerPolicy -> registerFunction Call